.travis.yml

```diff
@@ -5,9 +5,20 @@ python:
     - 3.4
     - 3.5
     - 3.6
+matrix:
+    include:
+        - python: "3.7"
+          dist: xenial
+          sudo: true
 install:
+    - pip install coveralls coverage
     - pip install tox-travis
+    - pip install zc.buildout
```

Comment:
Please don't introduce buildout into Travis when it wasn't there before.

I'd rather see something simpler like most projects have:
```yaml
install:
    - pip install -U coveralls coverage
    - pip install -U -e .[test]
script:
    - coverage run -m zope.testrunner --test-path=src
```

Reply:
Okay, i won't. is there a reason why in some projects there is buildout in travis and in some not?

Reply:
If not, i would throw buildout out of those projects I'm seeing now.

Reply:
There are a few projects (ZODB was one, IIRC) where having a buildout on CI was useful, but most projects are much simpler than that and don't require it. I tend to remove it when I come across it if it's not needed. I do this because it's an extra moving piece that can break and it's usually overkill for what we're trying to accomplish, so it just adds overhead.

Reply:
Seems reasonable. Thank you! I corrected it.